.lefthook.yml

```
@@ -0,0 +1,21 @@
+      glob: ".github/workflows/*.{yml,yaml}"
+      run: actionlint {staged_files}
+    editorconfig-checker:
+      run: editorconfig-checker {staged_files}
+    aicfg-link:
+      glob:
+        - CLAUDE.md
+        - AGENTS.md
+        - .claude/**
+        - .agents/**
+      run: uvx aicfg link agents --to claude-code --ci
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)